services/web/src/index.css

```diff
@@ -72,7 +72,7 @@ button:focus-visible {
 }
```

Comment:
I assume these are linter changes. Investigate if everything is ok after your changes.